apps/api/src/domain/models/user-management/user/invariant-validation/validateCoscradUser.spec.ts

```diff
@@ -71,7 +71,11 @@ describe('validateCoscradUser', () => {
                                 propertyName
                             );
 
-                            const result = validateCoscradUser(invalidDto);
+                            const invalidInstance = new CoscradUser(
+                                invalidDto as unknown as DTO<CoscradUser>
```

Comment:
It's too bad we have to make an invalid instance in order to validate invariants with our new approach, but I think being able to use a decorator to mixin the `class-validator` `simple invariant` errors is worth it!